sentry-android-integration-tests/sentry-uitest-android/src/androidTest/java/io/sentry/uitest/android/EnvelopeTests.kt

```diff
@@ -174,6 +177,12 @@ class EnvelopeTests : BaseUiTest() {
 
     @Test
     fun checkTimedOutProfile() {
+        Assume.assumeFalse(
```

Comment:
did you try with the latest changes from master, too?

Reply:
you mean your fixes for the envelopes with 0-byte bodies? yeah I tried that, still happening